fraudDetect-Jessie.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import division, print_function, unicode_literals
+
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler, Normalizer
+from sklearn.naive_bayes import BernoulliNB
+from sklearn.linear_model import LogisticRegression
+from sklearn.linear_model import SGDClassifier, Perceptron
+from sklearn.neural_network import MLPClassifier
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.ensemble import RandomForestClassifier, AdaBoostClassifier
+from sklearn.ensemble import GradientBoostingClassifier
+from sklearn.ensemble import BaggingClassifier
+from sklearn.svm import SVC
+import pandas as pd
+import time
+
+datapath = "/Users/jessiefan/Downloads/"
```

Comment:
This won't work for anyone else since the path is different. Add arguments to the file, if you need to see how look at the activity_producer.py file in the  feature/activity_producer branch